src/efs/efs3.py

```diff
@@ -0,0 +1,58 @@
+"""Eatfirst FileSystem for S3."""
```

Comment:
💄 "EatFirst", capital "F".

There are other wrong comments with this spelling.